plugin.ini

```diff
@@ -7,6 +7,6 @@ support_link="https://github.com/scholarslab/Neatline/issues"
 license="Apache License, Version 2.0"
 omeka_minimum_version="2.0"
 omeka_target_version="2.5"
```

Comment:
We might double check these Omeka versions. I believe Omeka 2.6 is now available, so we might target that release, and it would be good to be sure about 2.0 as a minimum version.

Reply:
I was incorrect; Omeka Classic is only up to 2.5.1; I would say this is good to merge and release. Thanks!